Packs/Ironscales/Integrations/Ironscales/Ironscales.py

```diff
@@ -0,0 +1,267 @@
+        outputs_key_field="incident_id",
+        outputs=incident,
+    )
+
+
+def classify_incident_command(client: Client, args: Dict[str, Any]) -> str:
+    incident_id = args.get("incident_id", None)
+    company_id = args.get("company_id", None)
+    classification = args.get("classification", None)
+    prev_classification = args.get("prev_classification", None)
+    email = args.get("email", None)
+
+    if not (incident_id or classification or prev_classification or email):
+        raise ValueError("Missing arguments!")
+
+    classify = client.classify_incident(
+        incident_id, classification, prev_classification, email, company_id
+    )
+
+    if classify:
+       return "Classification Succeeded!"
+    else:
+        return "Classification Failed!"
+
+
+def get_open_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    company_id = args.get("company_id", None)
+
+    open_incidents = client.get_open_incidents(company_id)
+    if open_incidents:
+        readable_output = tableToMarkdown(f"Open Incidents:", open_incidents)
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix="Ironscales.OpenIncidents",
+            outputs_key_field="incident_ids",
+            outputs=open_incidents,
+        )
+
+
+def test_module(client: Client, api_key, scopes) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: HelloWorld client to use
+
+    :type name: ``str``
+    :param name: name to append to the 'Hello' string
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    try:
+        client.get_jwt_token(api_key, scopes)
+    except DemistoException as e:
+        if "FORBIDDEN" in str(e):
+            return "Authorization Error: make sure API Key is correctly set"
+        else:
+            raise e
+    return "ok"
+
+
+def fetch_incidents(client: Client, last_run: Dict[str, Any], first_fetch: set, api_key: str, scopes: list) -> List:
+    last_run = last_run.get("data", None)
+    if last_run is None:
+        last_run = first_fetch
+    else:
+        last_run = set(last_run)
+
+    incidents_to_create = []
+    new_incidents = client.get_open_incidents()["incident_ids"]
+    if not new_incidents:
+        new_incidents = set()
+
+    incidents = []
+    incidents = set(new_incidents) - last_run
+
+    for incident in incidents:
+        data = client.get_incident(incident)
+        json_data = json.dumps(data)
+        incident_name = "Ironscales incident : IS-%s" % incident
+        incident = {
+            "name": incident_name,
+            "occurred": json_data.get("first_reported_date"),
+            "rawJSON": json_data,
+            'type': 'Ironscales'
```

Comment:
As discussed in the demo, this should be removed since the user chooses the incident type in the instance configuration window.
```suggestion
```